sympy/solvers/ode/ode.py

```diff
@@ -5901,4 +5773,5 @@ def _nonlinear_3eq_order1_type5(x, y, z, t, eq):
         Bernoulli, SingleODEProblem, SingleODESolver, RiccatiSpecial,
         SecondNonlinearAutonomousConserved, FirstExact, Liouville, Separable,
         SeparableReduced, HomogeneousCoeffSubsDepDivIndep, HomogeneousCoeffSubsIndepDivDep,
-        HomogeneousCoeffBest, LinearCoefficients, NthOrderReducible, Hypergeometric2nd)
+        HomogeneousCoeffBest, LinearCoefficients, NthOrderReducible, Hypergeometric2nd,
+        NthConstCoeffHomogen, NthConstCoeffVar, NthConstCoeffUndet)
```

Comment:
We should keep the naming consistent. None of the other solvers has abbreviations like `Undet`. Also the other class names correspond directly to the hint names except in camel case rather than snake case.

Reply:
`NthLinearConstantCoeffUndeterminedCoefficients`  but wouldn't it be too long as a class name?

Reply:
Hardly anyone ever needs to type the class name. The hint is what users have to type and is `nth_linear_constant_coeff_undetermined_coefficients`. It would have been good to make that shorter but it's too late for that now. For the class names consistency is more important than length.